packages/backend-core/src/context/types.ts

```diff
@@ -6,4 +6,5 @@ export type ContextMap = {
   appId?: string
   identity?: IdentityContext
   environmentVariables?: Record<string, string>
+  scimCall?: boolean
```

Comment:
Not sure if this is the best name for it. Any ideas to make something more explicit, or is it clear enough?

Reply:
adding `is` in front might help like `isScimCall` or just `isScim`